thirdparty/jira.go

```diff
@@ -183,7 +183,7 @@ func (jiraHandler *JiraHandler) GetJIRATicket(key string) (*JiraTicket, error) {
 // JQLSearch runs the given JQL query against the given jira instance and returns
 // the results in a JiraSearchResults
 func (jiraHandler *JiraHandler) JQLSearch(query string, startAt, maxResults int) (*JiraSearchResults, error) {
-	apiEndpoint := fmt.Sprintf("https://%v/rest/api/latest/search?jql=%v&startAt=%d&maxResults=%d", jiraHandler.JiraServer, url.QueryEscape(query), startAt, maxResults)
+	apiEndpoint := fmt.Sprintf("%v/rest/api/latest/search?jql=%v&startAt=%d&maxResults=%d", jiraHandler.JiraServer, url.QueryEscape(query), startAt, maxResults)
```

Comment:
I know this isn't related to your change, but it's worth changing `%v` to `%s` when we can.